Makefile

```
@@ -107,7 +107,10 @@ bottles/openssl:
 bottles/pcre: bottles/openssl
 	./scripts/fetch-brew-bottle.sh pcre
 
-bottles: bottles/openssl bottles/pcre
+bottles/hunspell: bottles/hunspell
```

```suggestion
bottles/hunspell: bottles/pcre
```

We want them to run one at a time even with e.g. `make -j16`, otherwise the overlapping invocations of `brew update` (made by `scripts/fetch-brew-bottle.sh`) will interfere each other.

So just make that change and it's good.

But, I wanted to share, based on some new learnings and experimentation just now, we can replace the existing bottles logic with this:
```Makefile
ifeq ($(detected_OS),Darwin)
BOTTLES_DIR := $(shell pwd)/bottles
BOTTLES := $(addprefix $(BOTTLES_DIR)/,hunspell openssl pcre)

$(BOTTLES): | $(BOTTLES_DIR)
	./scripts/fetch-brew-bottle.sh $(notdir $@)

$(BOTTLES_DIR):
	mkdir -p $(BOTTLES_DIR)
	brew update

bottles: $(BOTTLES)
endif
```

It will be easier to maintain, just add a needed bottle to the `BOTTLES :=` list.

But we'll also need to change `scripts/fetch-brew-bottle.sh` so that it doesn't run `brew update`, so I'll introduce  the new approach in a separate small PR after this PR is merged.

---

I just realized the code I commented on was part of #3463 (already merged). Even so, it needs to be changed.